packages/@aws-cdk/aws-lambda-nodejs/lib/bundling.ts

```diff
@@ -174,7 +179,7 @@ export class Bundling implements cdk.BundlingOptions {
       osPlatform,
     });
     const environment = this.props.environment ?? {};
-    const cwd = path.dirname(this.props.depsLockFilePath);
+    const cwd = this.projectRoot;
```

Comment:
```suggestion
    const cwd = this.props.projectRoot;
```

Reply:
@jogold are you sure about this suggestion? Doesn't it make more sense to use `this.projectRoot`?

Reply:
we have `this.projectRoot = props.projectRoot` but now that `projectRoot` is a mandatory prop we can use `this.props.projectRoot` everywhere.

Reply:
Oh, this is for the bundling... Didn't realize.

Still feels like a good practice to use the private member field if it's defined.